src/pages/index.js

```diff
@@ -1,257 +1,303 @@
-          description:
-            'Docs are organized in the service catalog and searchable for discoverability.',
-          icon: SearchCircleIcon,
-        },
-        {
-          name: 'Demonstrated at scale',
-          description:
-            'Spotify experienced an explosion of docs writing after adopting this methodology internally.',
-          icon: PresentationChartLineIcon,
-        },
+      title: 'Make your Developer Portal truly yours',
+      description: 'Roadie lets you install private plugins and renderers',
+      illustration: {
+        png: CustomPluginsImg,
+        alt: '',
+      },
+      paragraphs: [
+        'Your Developer Portal will only be successful if it’s tailored to the way your developers work. Roadie lets you bring your own Backstage plugins so you can integrate internal systems into your Developer Portal.',
+        'Roadie also lets you bring your own API documentation renderer so your docs are presented exactly as you want them inside Backstage.',
       ],
     },
     {
-      title: 'Track engineering maturity and migrations',
-      description:
-        'Define engineering standards and initiatives that get checked automatically across your ecosystem.',
-      prompt: 'Coming soon',
-      bullets: [
-        {
-          name: 'Find out which teams are experiencing challenges',
-          description:
-            'Standards adherence can be grouped by teams, offering insight into who may need more support to adopt new practices.',
-          icon: UserGroupIcon,
-        },
-        {
-          name: 'Understand your migration’s progress',
-          description:
-            'Initiatives are designed to keep track of wide-org migrations, offering insights about the change across software components. ',
-          icon: ChartBarIcon,
-        },
+      title: 'Integrate with your Kubernetes Clusters',
+      description: 'Roadie lets you connect to your infrastructure using a broker ',
+      illustration: {
+        png: KubernetesImg,
+        alt: '',
+      },
+      paragraphs: [
+        'Improving your microservices discoverability is what Roadie excels at, and that includes how and where they are deployed. Roadie provides a Kubernetes view catered for developers so they understand how their services are running and can optimize from there.',
+        'To integrate with your internal APIs, including your Kubernetes clusters, Roadie relies on a Broker model, which allows us to access your endpoints without exposing them to the public internet. This is the same approach used by Snyk, the security monitoring company.',
       ],
     },
   ],
 };
 
-const PLUGINS_SUPPORTED = {
-  title: 'Powerful plugins',
-  subTitle: `Open-source Backstage plugins are supported by default. Push your own if you need something bespoke.`,
-  link: {
-    text: 'Browse our plugins and integrations catalog',
-    to: '/docs/integrations/',
-  },
+// const SOLUTIONS = {
```

Comment:
I think we should leave these content blocks in the design until they each have their own pages. Otherwise we are cutting useful information that helps people understand the use cases of Roadie Backstage.